app/scripts/modules/core/src/pipeline/config/stages/wercker/werckerStage.module.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const angular = require('angular');
+
+import { STAGE_CORE_MODULE } from '../core/stage.core.module';
+import { TIME_FORMATTERS } from 'core/utils/timeFormatters';
+
+module.exports = angular.module('spinnaker.core.pipeline.stage.wercker', [
+  require('./werckerStage.js').name,
```

Comment:
This branch doesn't currently build for me locally and it looks like TravisCI is having trouble as well.  I believe the reason is that these require statements don't need to include the file extension.  Using output from the `yarn tslint` and `yarn build` commands as a guide can help in tracking down issues.

Reply:
fixed some broken tests and file name, yarn run test should be good now.